notebooks/05_operationalize/sar_webservice_poc.ipynb

```diff
@@ -0,0 +1,886 @@
+{
```

Comment:
<p>the training part is pretty similar to <a href="https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movielens_with_azureml.ipynb" target="_blank">https://github.com/microsoft/recommenders/blob/master/notebooks/00_quick_start/sar_movielens_with_azureml.ipynb</a> maybe we should consider to split these two notebooks into two, one for training and one for o16n. The training could go in the quickstart or model folder and the o16n here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Yeah I'm not sure the best way to structure this. Each of these are trying to show the complete steps needed to produce models for the mobile app. It may be that it is better to host these notebooks there?
I think separating them will make it hard to use and introduce a chance for the notebooks to get out of sync.